cukinia

```diff
@@ -69,7 +69,17 @@ _cukinia_result()
 {
 	local result="$1"
 
-	[ "$result" = "PASS" -o "$result" = "FAIL" ] || return 1
+	case "$result" in
+	PASS|pass)
```

Comment:
Personnaly, I think we should stick to upper or lower case.

Or use ${result^^} or ${result,,}